ocp_resources/cluster_role_binding.py

```diff
@@ -9,3 +10,28 @@ class ClusterRoleBinding(Resource):
     """
 
     api_group = Resource.ApiGroup.RBAC_AUTHORIZATION_K8S_IO
+
+    def __init__(
+        self,
+        name=None,
+        cluster_role=None,
+        subjects=None,
+    ):
+        super().__init__(name=name)
+        self.cluster_role = cluster_role
+        self.subjects = subjects
+
+    def to_dict(self):
+
+        self.res = super().to_dict()
+        self.res.setdefault("roleRef", {})
```

Comment:
`self.res.setdefault("roleRef", {})`
Is this always needed? if not move it under the if.